CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 # add a directory for cmake modules
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
-
+set(CMAKE_CXX_COMPILER g++-6)
```

Comment:
Don't do that here. It breaks build tests. Please use `CC=gcc-6 CXX=g++-6 cmake (all other options)`